src/components/GlobalPreferences/GlobalPreferences.js

```diff
@@ -98,13 +104,30 @@ function GlobalPreferences({
         />
       ) : (
         <React.Fragment>
-          <Tabs
-            items={VALUES.filter(
-              (_, index) => !!wrapper || index === NETWORK_INDEX
-            )}
-            onChange={onNavigation}
-            selected={sectionIndex}
-          />
+          {tabItems.length > 1 ? (
+            <Tabs
+              items={tabItems}
+              onChange={onNavigation}
+              selected={sectionIndex}
```

Comment:
When there is only 1 item this `sectionIndex` was not being set correctly, no need for the other changes if we set this correctly.